src/package.json

```diff
@@ -39,7 +39,6 @@
     "@dpc-sdp/ripple-link": "1.4.0",
     "@dpc-sdp/ripple-list": "1.4.0",
     "@dpc-sdp/ripple-list-group": "1.4.0",
-    "@dpc-sdp/ripple-map": "1.4.0",
```

Comment:
Hey @vincentdigital We also need to remove it from `yarn.lock` file. Run `yarn install` should able to update it.